Author: Pino Toscano <pino@kde.org>
Description: remove extra bool parameter for KConfigGroup::deleteEntry()
 unneeded, and it does not match any of the actual signatures of the
 function
Bug-Debian: https://bugs.debian.org/811679
Applied-Upstream: commit:1452683

--- a/src/kvpncconfig.cpp
+++ b/src/kvpncconfig.cpp
@@ -4305,7 +4305,7 @@
     appPointer->processEvents();
 
     if (useKwallet && KWallet::Wallet::isEnabled())
-        configgroup.deleteEntry("First use of Kwallet", false);
+        configgroup.deleteEntry("First use of Kwallet");
 
 
     /* = user data = */
Author: Pino Toscano <pino@kde.org>
Description: do not return bool for QString
 GCC 6 cannot convert them
Bug-Debian: https://bugs.debian.org/811679
Applied-Upstream: commit:1452682

--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -914,7 +914,7 @@
     NameAndPidOfProgramListenProcess->start(proc, args);
     if (!NameAndPidOfProgramListenProcess->waitForStarted()) {
         kError() << "netstat fails!" << endl;
-        return false;
+        return "";
     } else {
         NameAndPidOfProgramListenProcess->waitForFinished();
 
@@ -946,7 +946,7 @@
     GetEmailAddressOfCertProcess->start(proc, args);
     if (!GetEmailAddressOfCertProcess->waitForStarted()) {
         kError() << "GetEmailAddressOfCertProcess" << endl;
-        return false;
+        return "";
     } else {
         GetEmailAddressOfCertProcess->waitForFinished();
         disconnect(GetEmailAddressOfCertProcess, SIGNAL(readyReadStandardOutput()), this, SLOT(readOutGetEmailAddressOfCert()));
@@ -1159,7 +1159,7 @@
     GetHostnameProcess->start(proc, args);
     if (!GetHostnameProcess->waitForStarted()) {
         kError() << "Unable to start getHostname process!" << endl;
-        return false;
+        return "";
     } else {
         GetHostnameProcess->waitForFinished();
         return Hostname;
